prefab/curl-ssl/README.md

```diff
@@ -1,12 +1,27 @@
 # curl-ssl
 
-The curl-ssl sample shows how to import native dependencies from Maven. This
-sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most
-recent 10 reviews submitted to AOSP's code review system.
+The curl-ssl sample shows how to use [Prefab] to import native dependencies from [Google Maven]. This  
+sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most  
+recent 10 reviews submitted to AOSP's code review system.  
+To use [Prefab] with Android Studio 4.0+, simply follow the 3 steps:
```

Comment:
Android Gradle Plugin, not Android Studio. Android Studio is not relevant to this sample.

Also, need an empty line to start a new paragraph.

Reply:
done